app/models/user.rb

```diff
@@ -66,6 +66,10 @@ def password_complexity
                           1 uppercase, 1 lowercase, 1 digit and 1 special character'
   end
 
+  def to_s
```

Comment:
are we using this anywhere 🤔

Reply:
yes. i am going to try to explain it, then i will find a link and it will explain it better, haha.
on any `ActiveRecord` model, if you put a `to_s` (which i think it short for "to string") method, when you use the mode like `user`, it just makes it short hand to display whatever is in the `to_s` method. in this case, the `email` is shown. but you could put `#{first_name} #{last_name}" and it would show the user's first name and last name when you simply put something like `practice.user`, instead of having to concatenate/interpolate every time.

Reply:
https://www.easyactiverecord.com/blog/2014/12/02/rails-models-and-the-to-s-method/

Reply:
ahhh ok! I remember reading about this but it is helpful to see this actually being used! thank you! :D